single-line-diagram-viewer/src/main/java/com/powsybl/sld/viewer/SingleLineDiagramViewer.java

```diff
@@ -138,6 +139,8 @@
     private final ComboBox<String> diagramNamesComboBox = new ComboBox<>();
 
     private class ContainerDiagramPane extends BorderPane {
+        private final String divId = UUID.randomUUID().toString();
+
         private final ScrollPane flowPane = new ScrollPane();
```

Comment:
I think you could remove this and replace it with the webview, so we don't create a new one each time a diagram is drawn.